.github/workflows/prcodecheck.yml

```diff
@@ -1,3 +1,17 @@
+# Copyright 2023 Klepsydra Technologies AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Quick question -- since License.md already has the apache license, is not easier to reference that file (like we did preivously) rather than having a link to the apache website everywhere?

Reply:
We got this header from the Apache 2.0 recommendations:
https://www.apache.org/legal/src-headers.html